### PR TITLE
Fi 339 kick off header tests

### PR DESCRIPTION
**Pull request by yunwwang**
Pull requests into Inferno require the following items to be completed. Submitter and reviewer 
should check the relevant check boxes when the associated item is done. For items that are not 
applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
Requests do not require ticket links.

**Submitter:**
- [x] This pull request describes why these changes were made
This pull request added two tests for invalid headers in export request
- [x] Internal ticket for this PR:
FI-339
- [x] Internal ticket links to this PR
https://oncprojectracking.healthit.gov/support/browse/FI-339
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code
- [x] Tests are included and test edge cases
- [x] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb [hunk 1]
```diff
@@ -15,14 +15,24 @@ class BulkDataPatientExportSequence < SequenceBase
...
+        reply = export_kick_off(klass: klass)
 
         assert_response_accepted(reply)
         @content_location = reply.response[:headers]['content-location']
 
         assert @content_location.present?, 'Export response header did not include "Content-Location"'
       end
 
+      def assert_export_kick_off_fail_invalid_accept(klass)
+        reply = export_kick_off(klass: klass, headers: { accept: 'application/fhir+xml', prefer: 'respond-async' })
```
Comment (Jammjammjamm) [inline lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb]: I don't think these assertions should have side effects. I think the assertions should just be asserting something rather than carrying out the entire test. These tests are simple enough that all this code could just live in the test.

Comment (yunwwang) [reply lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb]: These are the test logic. I separate these logic from the test step, i.e. "test 'Server shall reject for $export operation with invalid Accept header'", so the logic can be easily unit tested. These are temporary solution before the test object approach

Comment (Jammjammjamm) [reply lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb]: It's fine to separate out that logic, but I don't think it belongs in an assertion. The assertion should just test the result of some previous steps, not perform the entire test.

Comment (yunwwang) [reply lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb]: I cannot using "test_" as prefix. Maybe I will rename to "check_"?

Comment (yunwwang) [reply lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb]: fixed
